packages/typespec-python/test/generated/lro-lro-core/azure/lro/core/models/_enums.py

```diff
@@ -0,0 +1,19 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from enum import Enum
+from azure.core import CaseInsensitiveEnumMeta
+
+
+class OperationState(str, Enum, metaclass=CaseInsensitiveEnumMeta):
```

Comment:
we shouldn't generate this right? is the code changes here resulting in this generation?

Reply:
This is a wrong committed file. I've removed it. It seems our CI will not check the num of generated files.